dev-python/vcrpy/vcrpy-4.2.1-r1.ebuild

```diff
@@ -27,7 +27,7 @@ RDEPEND="
 	dev-python/pyyaml[${PYTHON_USEDEP}]
 	dev-python/requests[${PYTHON_USEDEP}]
 	>=dev-python/six-1.5[${PYTHON_USEDEP}]
-	dev-python/urllib3[${PYTHON_USEDEP}]
```

Comment:
We don't do this unless there's a bug reference, or it'll end up rotting there forever.

Reply:
good now?